packages/client/src/stores/ReportStore.ts

```diff
@@ -103,8 +103,17 @@ export default class ReportStore {
     this.scapeExecutionStatusResponse = scapeExecutionStatusResponse;
     this.scapeResults = scapeExecutionStatusResponse.canaryAnalysisExecutionResult as CanaryAnalysisExecutionResult;
     this.scapeExecutionRequest = scapeExecutionStatusResponse.canaryAnalysisExecutionRequest as CanaryAnalysisExecutionRequest;
-    this.startTime = scapeExecutionStatusResponse.startTimeIso as string;
-    this.endTime = scapeExecutionStatusResponse.endTimeIso as string;
+
+    if (scapeExecutionStatusResponse.startTimeIso != null) {
```

Comment:
What happens if this is null?

Reply:
These null checks for `startTimeIso` and `endTimeIso` were added in because `safeGet()` didn't want a string that was potentially undefined. There might be a better or cleaner way to do this.